libsrc/c64/soft80_conio.s

```diff
@@ -67,22 +67,21 @@ firstinit:
 
         inc     soft80_first_init
 
+        ; save 4 bytes due to soft80_lo_charset and soft80_hi_charset being
+        ; page-aligned.
```

Comment:
Sorry for the nit picking but please shorten this comment to 
; soft80_lo_charset and soft80_hi_charset are page-aligned